### DISCUSSION TITLE
2023/03/20/lenovo-t14s-gen3-amd/

### DISCUSSION THREAD

**Discussion by giscus** [jdeluyck/kcoreorg-website#149] [Blog Comments]
# 2023/03/20/lenovo-t14s-gen3-amd/

10 years ago I bought a Dell XPS13 L322x ultrabook as a replacement for my white Macbook 2,1. This week I replaced the Dell with something newer: a Lenovo Thinkpad T14s Gen3 (AMD).

https://kcore.org/2023/03/20/lenovo-t14s-gen3-amd/

**Comment by pkarnadhar** [comment]
Having explored technical questions regarding the Lenovo ThinkPad T14s Gen3 (AMD), I'm increasingly convinced that it's a laptop worth considering for purchase. The depth of technical inquiries has shed light on its capabilities, particularly its compatibility with Linux OS, a crucial factor for my usage. The detailed information obtained here is reassuring and strengthens my confidence in the laptop's performance and suitability for my needs.
